grails-app/controllers/org/pih/warehouse/picklist/ImportPickCommand.groovy

```diff
@@ -11,7 +11,9 @@ class ImportPickCommand implements Validateable {
     String lotNumber
     Date expirationDate
     String binLocation
-    Integer quantity
+    String quantityAsText
+
+    static transients = ['quantity']
```

Comment:
fyi: in Grails 3 we don't need to declare a transient anymore. The getter is enough.

Reply:
I think I prefer transients over setters but we can discuss.

Reply:
We don't need a setter neither. Since Grails 2.0 we can declare a getter without having to introduce a transient field:
https://docs.grails.org/6.1.2/ref/Domain%20Classes/transients.html

> As of Grails 2.0 if there is only a getter or only a setter method, you don’t need to declare the property name of the method in the transients list. Only typed fields and get/set pairs that form a property but shouldn’t be persisted need to go in the transients list.